packages/wrangler/e2e/secrets-store.test.ts

```diff
@@ -0,0 +1,193 @@
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { readConfig } from "../src/config";
+import {
+	deleteStore,
+	listSecrets,
+	listStores,
+} from "../src/secrets-store/client";
```

Comment:
E2E tests shouldn't import directly from Wrangler's source. They're intended to test things as a user would (i.e. just by running the Wrangler CLI)